scripts/lib/setup-apt-repo

```diff
@@ -4,6 +4,12 @@ set -e
 
 SOURCES_FILE=/etc/apt/sources.list.d/zulip.list
 STAMP_FILE=/etc/apt/sources.list.d/zulip.list.apt-update-in-progress
+SUCCESS_FILE=/etc/apt/sources.list.d/zulip.list.success
```

Comment:
This doesn't solve the "files changed" problem.  As I said, I think we want to hash this file + `scripts/setup/*.asc`